pkgs/servers/mail/archiveopteryx/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, fetchurl, openssl, perl, zlib }:
+with builtins;
+let
+  version = "3.2.0";
+in stdenv.mkDerivation {
+  name = "${baseNameOf ./.}-${version}";
```

Comment:
- dont rely on baseNameOf and just type name (this makes expression not depending in which directory it is)
- you can use ``rec`` and drop ``let ... in`` and also use ``${name}`` in the url